controller/src/controller/config.py

```diff
@@ -527,6 +582,120 @@ class PrepullerConfig(PrepullerOptions):
     source: DockerSourceConfig | GARSourceConfig
 
 
+class IndividualImageClassPolicy(BaseModel):
```

Comment:
I don't think the `Individual` part of this class name is doing much for clarity other than making the class name longer. Maybe just `ImageFilterPolicy` instead? I think that's what this is.

Reply:
This one is ImageFilterPolicy and the enclosing one is RSPImageFilterPolicy.  I think that's pretty clear.  I also added an alias attribute to RSPImageFilterPolicy, and the reason why is so that I don't have to special case aliases on the implementation side.